bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -998,9 +1008,21 @@ ErrorOr<LBREntry> DataAggregator::parseLBREntry() {
   if (std::error_code EC = MispredStrRes.getError())
     return EC;
   StringRef MispredStr = MispredStrRes.get();
-  if (MispredStr.size() != 1 ||
-      (MispredStr[0] != 'P' && MispredStr[0] != 'M' && MispredStr[0] != '-')) {
-    reportError("expected single char for mispred bit");
+  // SPE brstack mispredicted flags might be two characters long: 'PN' or 'MN'.
```

Comment:
nit:
```suggestion
  // SPE brstack mispredicted flags might be up to two characters long: 'PN' or 'MN'.
```


Can add that 'N' is optional.

Reply:
Done.